jenkins/e2e-image/e2e-runner.sh

```diff
@@ -47,10 +47,11 @@ e2e_go_args=( \
   --dump="${ARTIFACTS}" \
 )
 
-# We get the Kubernetes tarballs unless we are going to use old ones
+# TODO(fejta): delete all of these soon, as RAW_EXTRACT is the only supported
+# option. Meaning call to e2e-runner.sh needs an --extract flag
 if [[ -n "${RAW_EXTRACT:-}" ]]; then
```

Comment:
Who's still using it? Seems we already have presubmit to ensure all e2e jobs has an --extract flag, but we still set RAW_EXTRACT on all the jobs?